### PR TITLE
Add Link : Fixes issue related to url prefix

### PR DESCRIPTION
**Pull request by kumarpalsinh25**
Fixes https://github.com/acterglobal/a3-meta/issues/510

- Removed predefine `https://` prefix
- Used more generic url validation pattern which can work with custom schema urls also

### CODE REVIEW

File: app/lib/common/utils/utils.dart [hunk 1]
```diff
@@ -36,8 +36,10 @@ final idMatrixRegexp = RegExp(
   r'matrix:roomid/(?<id>[^?]+)(\?via=(?<server_name>[^&]+))?(&via=(?<server_name2>[^&]+))?(&via=(?<server_name3>[^&]+))?',
 );
 
-final urlValidatorRegexp = RegExp(
```
Comment (gnunicorn) [inline app/lib/common/utils/utils.dart:39]: and the regexp grows, yet now tests are nowhere to be found ...

how are we sure we don't break stuff, if we don't have tests for this?

**Review by gnunicorn — COMMENTED** [review]
I have no idea ... the fix ix primarily about a regexp and has no tests ...

File: app/test/features/general/url_validation_test.dart [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import 'package:acter/common/utils/utils.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
```
Comment (kumarpalsinh25) [inline app/test/features/general/url_validation_test.dart:4]: @gnunicorn 
Added UNIT TEST for Regex validation. All Test are passed with given regex!

**Review by gnunicorn — APPROVED** [review]
neat, thanks!

**Pull request merged**